lib/entice/logic/MapInstance.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Entice.Logic.MapInstance do
+  alias Entice.Entity
+  alias Entice.Logic.MapInstance
+
+
+  defstruct(players: %{},
+    npcs: %{},
+    map: nil)
+
+  def start_instance(map, player_entities \\ [], npc_info \\ []) do
+    {:ok, id, _pid} = Entity.start()
+    npc_entities = for _npc = %{name: name, model: model} <- npc_info,
```

Comment:
put this on a new indented line, its much easier to figure out that the do belongs to it as well that way :)